File: lib-fast4Android-core/src/main/java/io/github/keep2iron/android/net/exception/NoDataException.java
package io.github.keep2iron.android.net.exception;

/**
 * @author keep2iron <a href="http://keep2iron.github.io">Contract me.</a>
 * @version 1.0
 * @since 2018/01/31 18:55
 */
public class NoDataException extends RuntimeException {

    public NoDataException() {
    }

    public NoDataException(String message) {
        super(message);
    }

    public NoDataException(String message, Throwable cause) {
        super(message, cause);
    }

    public NoDataException(Throwable cause) {
        super(cause);
    }
}

File: lib-fast4Android-core/src/main/java/io/github/keep2iron/android/net/exception/NoDataException.kt
package io.github.keep2iron.android.net.exception

/**
 * @author keep2iron [Contract me.](http://keep2iron.github.io)
 * @version 1.0
 * @since 2018/01/31 18:55
 *
 * 无数据异常
 */
class NoDataException : RuntimeException {

    constructor() {}

    constructor(message: String) : super(message) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}

    constructor(cause: Throwable) : super(cause) {}
}
